chainer/_sphinx_ext.py

```diff
@@ -0,0 +1,40 @@
+import sphinx
+
+
+def qualify_name(attr_name, klass):
+    if klass and not attr_name.startswith('~'):
+        try:
+            q = klass.__qualname__
+        except AttributeError:
+            q = klass.__name__
+        return '~%s.%s' % (q, attr_name)
+    return attr_name
+
+
+def setup(app):
+    app.setup_extension('sphinx.ext.napoleon')
+
+    def _parse_attributes_section(self, section):
```

Comment:
I haven't looked into the details yet, but could you write down some comment about the difference to the original `_parse_attributes_section` function?